app/models/emoji.rb

```diff
@@ -60,6 +60,7 @@ def self.clear_cache
     Discourse.cache.delete(cache_key("standard_emojis"))
     Discourse.cache.delete(cache_key("aliases_emojis"))
     Discourse.cache.delete(cache_key("all_emojis"))
+    Discourse.cache.delete(cache_key("tonable_emojis"))
```

Comment:
what's your opinion on this ? To avoid duplicating all the key names, what should we do? Something like this ?

```ruby
CACHE_KEYS = {
  standard_emojis: "standard_emojis",
  aliases_emojis: "aliases_emojis",
  all_emojis: "all_emojis",
  tonable_emojis: "tonable_emojis"
}

def self.clear_cache
  CACHE_KEYS.values.each { |value| Discourse.cache.delete(cache_key(value)) }
end

def self.tonable_emojis
  Discourse.cache.fetch(cache_key(CACHE_KEYS[:tonable_emojis])) { db['tonableEmojis'] }
end

###
```